src/foam/comics/v2/DAOBrowserView.js

```diff
@@ -223,6 +223,10 @@ foam.CLASS({
       label: '',
       toolTip: 'Export Table Data',
       icon: 'images/export-arrow-icon.svg',
+      isAvailable: async function() {
+        var records = await this.__subContext__.exportDriverRegistryDAO.select();
```

Comment:
Why using __subContext__ instead of __context__, or better, import: 'exportDriverRegistryDAO', and then do this.exportDriverRegistryDAO?

Reply:
fixed